Python/formatter_unicode.c

```
@@ -979,12 +980,6 @@ format_long_internal(PyObject *value, const InternalFormatSpec *format,
        from a hard-code pseudo-locale */
     LocaleInfo locale = LocaleInfo_STATIC_INIT;
 
-    /* no precision allowed on integers */
```

Should this be kept for the `c` character type? Currently one can do:
```py
f"{65:.5c}" # 'A'
f"{3486:.100c}" # 'ඞ'
```
which silently skips the precision. If this is unintended perhaps it should be moved into the `/* special case for character formatting */`?